src/components/NavBar/NavBar.css

```diff
@@ -62,3 +63,35 @@
 #app-navbar .navbar-nav .navbar-btn.active .navbar-btn-legend {
   color: rgb(0, 225, 250);
 }
+
+#app-navbar #navbar-sponsor-logo {
+  max-width: 100px;
+  float: right;
+  margin-top: -5%;
+}
+
+#app-navbar #navbar-legend-div {
+  display: none;
+}
+
+#app-navbar #navbar-legend-div1 {
+  color: #888B8D;
+  height: 48px;
+  vertical-align: middle;
+  display: inline-block;
+  position: absolute;
+  right: 2px;
+}
+
+@media all and (max-width: 991px){
```

Comment:
#navbar-legend-div1 has almost the same styling as #navbar-legend-div, why don't you try using the same styling for both of them? and instead, use `@media all and (min-width)` to change position and right.